daemon/main.c

```diff
@@ -584,6 +584,14 @@ static void get_netdata_configured_variables() {
     // ------------------------------------------------------------------------
     // get default Database Engine page cache size in MiB
 
+    default_rrdeng_page_fetch_timeout = (int) config_get_number(CONFIG_SECTION_GLOBAL, "dbengine page fetch timeout", PAGE_CACHE_FETCH_WAIT_TIMEOUT);
+    if (default_rrdeng_page_fetch_timeout < 1)
```

Comment:
Considering that a value smaller than 1 is not accepted, should we write an error inside our logs giving conditions for users to change their configuration file?

Reply:
> should we write an error 

"info" please!

Reply:
I do not have objections to write an `info` instead `error`. :handshake:

Reply:
Ok added -- also moved the code until after logfiles are initialized